.env.example

```diff
@@ -1,9 +1,25 @@
-BACKEND_DEBUG=0   # 0|1
-POSTGRES_PASSWORD=
-POSTGRES_USER=
 DB_HOST=db
 DB_NAME=postgres
-FRONTEND_TARGET=production # development|production
-NODE_IMAGE=lts-alpine # lts-alpine|16.15-alpine|...
-JWT_SECRET_KEY=
-DISABLE_AUTHENTICATION=false # true|false
+# Credentials for the database host and the database.
+
+POSTGRES_PASSWORD=password
+POSTGRES_USER=postgres
+# Credentials for the database user.
+
+BACKEND_DEBUG=0
+# 0 : disable debug, useful for production.
+# 1 : enable debug, useful for development, backend will refresh everytime you modify the source code.
+
+FRONTEND_TARGET=production
+# development : faster refresh time when modification occurs.
+# production : to deploy the website.
+
+NODE_IMAGE=lts-alpine
+# Select which node's image will be used (eg: lts-alpine|16.15-alpine)
+
+JWT_SECRET_KEY=secret
+# The secret key is used to encrypt your JWT. For production purposes, please generate a secret 32 characters minimum and secure
+
+DISABLE_AUTHENTICATION=false
+# true : Authentication is disabled.
+# false : Authentication is enabled.
```

Comment:
```suggestion
```
I think too much documentation kills the documentation. Feel free to ignore this suggestion.